draft-ietf-rats-corim.md

```diff
@@ -1054,6 +1054,19 @@ The types or associating addressing information to a measured environment are:
 
 ##### Crypto Keys {#sec-crypto-keys}
 
+A `$crypto-key-type-choice` statement identifies a cryptographic key.
+There are several possible formats.
+The usage context helps determine how Verifiers process each variant.
+For example, as a Reference Value, the Verifier applies an Evidence matching algorithm.
+As an identity triple, the Verifier may need to obtain a suitable credential to authenticate the Attester.
+
+Additionally, the deployment context may add new processing requirements, such as, proof of possession checking,
+credential status checking, or distribution of various processing steps.
+For example, in a split Verifier scenario, a first Verifier may challenge the key to sign a nonce
+then compute a digest of the public key that is forwarded to a second Verifier.
+The second Verifier completes verification by locating the certificate,
+performing certificate path validation, revocation, and trust anchor checks.
```

Comment:
I'd drop this para entirely as it introduces too much variability without effectively specifying any particular behaviour.  Either that or leave just the bare minimum, i.e.: _"Additionally, the deployment context may add further processing requirements.  These are outside the scope of the present specification."_